lhotse/dataset/sampling/povey.py

```diff
@@ -0,0 +1,274 @@
+import math
+import os
+import random
+from pathlib import Path
+from typing import Dict, Generator, Iterable, List, Optional, Sequence, Tuple, Union
+
+import torch
+import torch.distributed as dist
+
+from lhotse import CutSet, Seconds
+from lhotse.cut.set import deserialize_cut
+from lhotse.dataset import DynamicBucketingSampler, DynamicCutSampler
+from lhotse.dataset.sampling.base import SamplingDiagnostics
+from lhotse.lazy import Dillable
+from lhotse.serialization import decode_json_line
+from lhotse.utils import Pathlike
+
+PathlikeAndScale = Tuple[Pathlike, float]
+
+
+class PoveySampler(torch.utils.data.Sampler, Dillable):
```

Comment:
There is a "povey window". 
Now we also have a "povey sampler"  😀

Reply:
LOL, but let's call it StatelessSampler.  With the Povey Window, I think people didn't get that it was basically a joke (since no-one really cares about things like windows these days), and they perhaps thought that it was a kind of self-aggrandizement.